lib/src/util/validation_util.dart

```diff
@@ -28,10 +28,16 @@ class ValidationUtil {
   /// Use this to log warnings to the console in dev mode only.
   /// This is to be used in assert calls for dev help only so that it gets
   /// compiled out for production.
+  ///
   ///     assert(ValidationUtil.warn('Some warning message'));
   ///
-  /// The message will get print out to the console.
-  static bool warn(String message) {
+  /// Optionally, a component can be passed as the second parameter
+  /// to provide additional information in the console.
+  ///
+  ///     assert(ValidationUtil.warn('Some warning message', component));
+  ///
+  /// The message will be printed out to the console.
+  static bool warn(String message, [dynamic component]) {
```

Comment:
Nope, you're right. The only case it'd be needed is if they're using plain react-dart component, but we don't really have to worry about that.